01 - Community/People/jdanielmourao.md

```diff
@@ -17,15 +17,15 @@ publish: true
 
 
 ## Author of
-
 %% Begin Hub: Released contributions %%
 
 ### Themes
 - [[Sanctum]]
 %% End Hub: Released contributions %%
 
 %% Add links to any plugins, themes or other notes that the author has created but are not (yet) included in the `obsidian-releases` repo %%
-
+### Guides
+- [[Want some Sass in your obsidian theme‽ here's How and Why]]
 <!--
```

Comment:
This comment line shows up in preview in Obsidian.

![image](https://user-images.githubusercontent.com/4840096/144303001-f211b598-0100-4253-b7fa-638adbe4b43d.png)

Changing this:

```
- [[Want some Sass in your obsidian theme‽ here's How and Why]]
<!--
```

to this:

```
- [[Want some Sass in your obsidian theme‽ here's How and Why]]

<!--
```

fixes it...

Reply:
Pushed the fix

Reply:
Yes, it's great - thanks.